docs/UserGuide.md

```diff
@@ -92,17 +101,78 @@ Adds a task to the planner.
 
    Default: No notes.
 
-#### Example usage 
+##### Command Format
+
+`add n/NAME [t/TIME] [d/DURATION] [D/DEADLINE] [r/RECURRENCE] [i/IMPORTANCE] [a/ADDITIONAL-NOTES]`
 
-* `add n/Assignment1 t/1100 D/16-09-2020 d/2 r/Today i/high a/Refer to lecture notes`
-* `add n/Assignment1 t/1100`
-* `a n/Assignment1 t/1100 D/16-09-2020 d/2 r/Today i/high a/Refer to lecture notes`
+##### Example Usage
 
-#### Expected output 
+The following shows the output from ATHENA after entering `add n/Assignment1 t/1100 D/16-09-2020 d/2 r/Today i/high a/Refer to lecture notes`.
+You should expect to see a message to confirm that the task is added.
 
-For first example usage `add n/Assignment1 t/1100 D/16-09-2020 d/2 r/Today i/high a/Refer to lecture notes`:
 ![add command screenshot](screenshots/athena%20add.jpg)
 
+#### Intermediate Usage
+
+The `add` command accepts 7 parameters.
+
+* `NAME` is the name of the task. It is a compulsory parameter the user has to input.
+
+The following parameters are optional. If they are left empty, ATHENA will use the default value assigned.
+* `TIME` is the time to start doing this task (**HHMM**). For example, 1100.
+
+   Default: Allocated by Athena (only available for non recurring tasks)
+* `DURATION` is the expected time taken to complete task (in hours).
+
+   Default: 1 hour.
+* `DEADLINE` is the date to do task by (**DD-MM-YYYY**). For example, 16-09-2020.
+
+   Default: No deadline.
+* `RECURRENCE` is one of **TODAY**, **MONDAY**, **TUESDAY**, **WEDNESDAY**, **THURSDAY**, **FRIDAY**, **SATURDAY**, **SUNDAY** or a specific date (**DD-MM-YYYY**).
+
+   This is where the user can input a specific date to do the task on. 
+   If it is not **TODAY** or a specific date (**DD-MM-YYYY**), it will be treated as occuring every week. For example, **MONDAY** means happening every Monday.
+   
+   Default: **TODAY**.
+* `IMPORTANCE` is one of **HIGH**, **MEDIUM**, **LOW**.
+
+   Default: **MEDIUM**.
+* `ADDITIONAL-NOTES` is the additional notes of the task.
+
+   Default: No notes.
+
+##### Command Format
+
+`add n/NAME [t/TIME] [d/DURATION] [D/DEADLINE] [r/RECURRENCE] [i/IMPORTANCE] [a/ADDITIONAL-NOTES]`
+
+##### Example Usage
+
+The following shows the output from ATHENA after entering `add n/Assignment2 t/1400 i/high`.
+You should expect to see a message to confirm that the task is added with some of the parameter's default value assigned.
+
+*TODO: Insert screenshot*
+
+#### Advanced Usage
+
+This section is an extension to the [Intermediate Usage](intermediate-usage) section.
```

Comment:
> `[Intermediate Usage](intermediate-usage)`

I think should be `[Intermediate Usage](#intermediate-usage)` but maybe not important we can fix the links later when we finish everything.